packages/now-static-build/src/index.ts

```diff
@@ -264,7 +264,6 @@ export async function build({
   const pkg = getPkg(entrypoint, workPath);
 
   const devScript = pkg ? getScriptName(pkg, 'dev', config) : null;
```

Comment:
Do we still need this local `getScriptName` function now that there is the one in build-utils?

Reply:
I'd prefer to leave it for now since the logic in this function is slightly different (changes behavior based on if it's zero-config or not).

Reply:
That's unfortunate. We still ended up with 3 different usages, one for each runtime 🤦‍♂️